src/controllers/ContinuedIllnessController.ts

```diff
@@ -14,7 +14,7 @@ import { Feature } from 'app/utils/Feature';
 import { CONTINUED_ILLNESS_PAGE_URI } from 'app/utils/Paths';
 import { Navigation } from 'app/utils/navigation/navigation';
 
-const template = 'continued-illness';
+const template = 'illness/continued-illness';
```

Comment:
the template is not the same thing as the URL. this should be the name of the file in the view directory. 

```suggestion
const template = 'continued-illness';
```

Reply:
Indeed. However, Natalie updated the structure of the `/views/` directory, presumably to neaten the whole thing up a bit, so I've followed suit.

Reply:
oh fancy

Reply:
Indeed! Might be nice to do the same to the controllers directory at a later stage, as that thing's getting looooong. But that's a story for another time.